_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -63,6 +63,16 @@ public static function register_endpoints() {
 		$module_data_endpoint = new Jetpack_Core_API_Module_Data_Endpoint();
 		$module_toggle_endpoint = new Jetpack_Core_API_Module_Toggle_Endpoint( new Jetpack_IXR_Client() );
 
+		register_rest_route( 'jetpack/v4', '/jitm', array(
+			'methods'  => WP_REST_Server::READABLE,
+			'callback' => __CLASS__ . '::get_jitm_message',
+		) );
+
+		register_rest_route( 'jetpack/v4', '/jitm', array(
+			'methods'  => WP_REST_Server::CREATABLE,
+			'callback' => __CLASS__ . '::delete_jitm_message'
```

Comment:
https://github.com/Automattic/jetpack/blob/7c33f35cba25b89cf777c65ac774a054ba3b362a/class.jetpack-jitm.php#L215-L215

Yes, it's on purpose. Instead of an error message, I just wanted to return an empty array. In fact, it should never return an error, only an array of jitms (if any).